### ISSUE TITLE
rental `bosminer` rig from rigly.io do not work when connected to tProxy

### ISSUE THREAD

**Issue by plebhash** [stratum-mining/stratum#964]
I was trying to direct some rented hashrate from [rigly.io](rigly.io) to a SV2 pool via translator proxy (tProxy).

tProxy logs:
```
2024-06-09T20:57:59.100108Z  INFO translator_sv2::lib::downstream_sv1::downstream: PROXY SERVER - ACCEPTING FROM DOWNSTREAM: 3.128.233.42:49016
2024-06-09T20:57:59.490776Z  INFO translator_sv2::lib::downstream_sv1::downstream: Down: Configuring
2024-06-09T20:57:59.899398Z ERROR translator_sv2::lib::status: Error: V1Protocol(Method(ParsingMethodError((WrongArgs(Array [String("2024-05-06-0-fb624381-24.04-plus;bosminer-plus-tuner 0.9.0-fb624381"), Null]), StandardRequest(StandardRequest { id: 1, method: "mining.subscribe", params: Array [String("2024-05-06-0-fb624381-24.04-plus;bosminer-plus-tuner 0.9.0-fb624381"), Null] })))))
2024-06-09T20:57:59.899492Z  WARN translator_sv2::lib::downstream_sv1::downstream: Downstream: Shutting down sv1 downstream reader
2024-06-09T20:57:59.899516Z  WARN translator_sv2::lib::downstream_sv1::downstream: Downstream: Shutting down sv1 downstream writer: 3.128.233.42:49016
2024-06-09T20:57:59.899648Z  INFO translator_sv2: HEALTHY message: V1 Protocol Error: `Method(ParsingMethodError((WrongArgs(Array [String("2024-05-06-0-fb624381-24.04-plus;bosminer-plus-tuner 0.9.0-fb624381"), Null]), StandardRequest(StandardRequest { id: 1, method: "mining.subscribe", params: Array [String("2024-05-06-0-fb624381-24.04-plus;bosminer-plus-tuner 0.9.0-fb624381"), Null] }))))
```

SV1 logs on rigly.io side (provided from their customer support)
```
2024-06-09 20:48:43:192:core_proxy.py: 91: DEBUG:139.59.2.141:33650:asyncio.open_connection with {‘upstream_host’: ‘185.130.45.51’, ‘upstream_port’: 34255, ‘upstream_user’: ‘x.Rigly005’, ‘proxy_id’: ‘102’, ‘proxy_port’: ‘3337’, ‘pid’: 13} pool_drop=0 client_info_str=‘139.59.2.141:33650’ semaphore._value=499

2024-06-09 20:48:43:440:core_proxy.py: 225: DEBUG:139.59.2.141:33650:Client 139.59.2.141:33650 -> Upstream: {“id”:0,“method”:“mining.configure”,“params”:[[“version-rolling”],{“version-rolling.mask”:“1fffe000”,“version-rolling.min-bit-count”:16}]}

2024-06-09 20:48:43:556:core_proxy.py: 388: INFO:139.59.2.141:33650:Upstream -> Client 139.59.2.141:33650: {“id”:0,“error”:null,“result”:{“minimum-difficulty”:false,“version-rolling”:true,“version-rolling.mask”:“1fffe000”,“version-rolling.min-bit-count”:“00000010”}}

2024-06-09 20:48:43:864:core_proxy.py: 225: DEBUG:139.59.2.141:33650:Client 139.59.2.141:33650 -> Upstream: {“id”:1,“method”:“mining.subscribe”,“params”:[“2024-05-06-0-fb624381-24.04-plus;bosminer-plus-tuner 0.9.0-fb624381”,null]}

2024-06-09 20:48:43:912:core_proxy.py: 225: DEBUG:139.59.2.141:33650:Client 139.59.2.141:33650 -> Upstream: {“id”:2,“method”:“mining.authorize”,“params”:[“MRR3337”,“”]}

2024-06-09 20:48:43:913:core_proxy.py: 300: DEBUG:139.59.2.141:33650:XNSUB not supported by miner machine. xnsub.state=<XnsubState.NOT_SUPPORTED: 2>

2024-06-09 20:48:43:913:core_proxy.py: 314: DEBUG:139.59.2.141:33650:Changing mining.authorize from MRR3337 to x.Rigly005.

2024-06-09 20:48:43:981:core_proxy.py: 373: INFO:139.59.2.141:33650:Pool closing connection ! counter=6

2024-06-09 20:48:43:981:core_proxy.py: 603: DEBUG:139.59.2.141:33650:process_connection END counter=6
```

**Comment by plebhash** [comment]
solved via #965

**Comment by plebhash** [comment]
solved via #965
